### PR TITLE
RFR: Allow rendering of non-string parameters in rule

### PR DESCRIPTION
**Pull request by lakshmi-kannan**
Refactored render_values used in action_chain to st2common to avoid code dupes. Improved tests.

### CODE REVIEW

File: st2common/st2common/util/jinja.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import json
```
Comment (Kami) [inline st2common/st2common/util/jinja.py]: Missing license header.

File: st2actions/st2actions/runners/actionchainrunner.py [hunk 2]
```diff
@@ -35,35 +32,13 @@
...
 from st2common.util import action_db as action_db_util
 from st2common.util import isotime
+from st2common.util import jinja as jinja_utils
 
 
 LOG = logging.getLogger(__name__)
 RESULTS_KEY = '__results'
 
 
-def render_values(values, context):
```
Comment (Kami) [inline st2actions/st2actions/runners/actionchainrunner.py]: Yay, much better for this to live in the utils.

File: st2common/st2common/util/jinja.py (hunk 1 above)
Comment (lakshmi-kannan) [inline st2common/st2common/util/jinja.py]: Oopsy dawg!

Comment (lakshmi-kannan) [inline st2common/st2common/util/jinja.py]: Also missed changelog :/

**Pull request merged**